### PR TITLE
[TS-1022]<fix>: fix alpine segmentation fault in startup

### PR DESCRIPTION
**Pull request by zitsen**

### CODE REVIEW

**Review by sangshuduo — APPROVED** [review]
LGTM

File: src/plugins/http/src/httpResp.c [hunk 1]
```diff
@@ -156,7 +156,10 @@ void httpSendErrorResp(HttpContext *pContext, int32_t errNo) {
 
   HttpServer *pServer = &tsHttpServer;
   SMonHttpStatus *httpStatus = monGetHttpStatusHashTableEntry(httpCode);
-  pServer->statusCodeErrs[httpStatus->index] += 1;
+  // FIXME(@huolinhe): I don't known why the errors index is overflowed, but fix it by index check
```
Comment (sangshuduo) [inline src/plugins/http/src/httpResp.c:159]: no need to fix in develop branch?

**Review by sangshuduo — COMMENTED** [review]
see my comments

File: src/plugins/http/src/httpResp.c (hunk 1 above)
Comment (zitsen) [reply src/plugins/http/src/httpResp.c:159]: needed. in pr https://github.com/taosdata/TDengine/pull/9522

**Review by afwerar — COMMENTED** [review]
LGTM

**Pull request merged**